apps/src/code-studio/components/AdvancedShareOptions.jsx

```diff
@@ -38,6 +38,12 @@ const style = {
     color: color.neutral_dark,
     fontFamily: "'Gotham 5r', sans-serif",
   },
+  warningp: {
```

Comment:
Could you explain why you added this to the PR? Thanks!

Reply:
Just looked at your other [open PR](https://github.com/code-dot-org/code-dot-org/pull/53106) and I think this change may belong to that one? If so, you can just revert [this commit](249d74f7b4bc2e15d8f925e2d5cd13a2cc704758).